### ISSUE TITLE
Ender Letter Crash on 2.2.2

### ISSUE THREAD

**Issue by Divineaspect** [lumien231/Random-Things#93]
Only posting this even though it's not for the most up to date version because there is no changelog to check against.

http://pastebin.com/hZ6rDbmj

**Comment by lumien231** [comment]
How is there no changelog to check against? The curseforge files page lists ALL versions with ALL changelogs

**Comment by Divineaspect** [comment]
Ok, if you want, it's not on your list of changes for 2.2.3

**Comment by lumien231** [comment]
Whoops o_O, i didnt mean to close this

**Comment by lumien231** [comment]
When does this occur?

**Comment by Divineaspect** [comment]
sorry for not including that, shift clicking with a new (without a target) ender letter.

**Comment by StarkRG** [comment]
I can confirm this happens for me as well. The first letter I create works fine, the second, however, causes this crash. After restarting the client the second and first letters both work fine, but if I open up a third one it causes a crash.

http://pastebin.com/m4i2QuHk

**Comment by lumien231** [comment]
Okay so i can't reproduce either of your problems. 
https://www.youtube.com/watch?v=7nqVq7JIKCM

Is that how it would crash for you?

**Comment by StarkRG** [comment]
Not quite, for me it happens after I've placed something in the first letter, in other words I'm preparing a bunch of letters to be addressed and sent at a later date.

**Comment by lumien231** [comment]
Okay i now did the same thing i did in the video except i filled the ender letters with item, i still don't crash. Can you reproduce this with only random things installed?

**Comment by ceselegend** [comment]
Same issue here, sending a first letter is perfectly working, but then opening a second letter to send later is crashing the game. The first one was including an item.

**Comment by Wyvern67** [comment]
I have it too. I first send a letter to a player with a single item inside and it works fine but if I send another one later it crashes upon opening the GUI.

**Comment by lumien231** [comment]
And you can reproduce that with only rt installed?

**Comment by StarkRG** [comment]
It's hard for me to test this with only RT as I don't have easy access to a dedicated server, but I will try.

**Comment by lumien231** [comment]
So it's not repeatable in single player?

**Comment by Divineaspect** [comment]
Can we even use Enderletters in single player?

**Comment by lumien231** [comment]
Yeah, you can send them to yourself.

**Comment by Divineaspect** [comment]
That's actually a surprisingly useful way to store items.

**Comment by Tearlow** [comment]
Thought I'd just leave this here as I have been having major issues with Ender letters myself. Other users can open the GUI perfectly fine but on my end a newly created letter crashes me with the same error as the OP's post.

After a bit of digging I notice it's the "tag > string:receiver" nbt (it needs to exist but doesn't contain any string) is what causing it for myself. If I add it manually beforehand it works perfectly fine!
